python/lib/communication/dmod/test/test_ngen_request.py

```diff
@@ -99,24 +138,6 @@ def test_to_dict_1_a(self):
         ex_dict = self.request_objs[example_index].to_dict()
         self.assertEqual(ex_dict, self.request_jsons[example_index])
 
-    def test_to_json_0_a(self):
```

Comment:
I believe I took these out for two reasons:
1. they broke, as expected, due to changes in _NGENRequest_'s attribute structure
2. I wasn't sure they could be reliable anyway, due to the possibility of JSON elements being re-ordered  

But, I just adjusted the implementation of `to_json` slightly to address item 2, so that output keys get sorted.  That should make things deterministic, and so I've re-added the tests and properly updated the test case setup.